cmd/goProbe/goProbe.go

```diff
@@ -82,6 +83,11 @@ func main() {
 		}()
 	}
 
+	// Set the release mode of GIN depending on the log level
+	if logging.LevelFromString(config.Logging.Level) != logging.LevelDebug {
```

Comment:
What if the API is disabled? Any potential side-effects / issues (because gin may not even be initialized)? We could also move the instantiation somewhere deeper to avoid `goProbe` / `global-query` now directly depend on `Gin` (e.g. in case we want to change that in the future again)...

Reply:
That's a good point. I'll make it an option for the two APIs (goProbe and global-query). Thanks